pandas/tests/extension/test_numpy.py

```diff
@@ -348,6 +348,12 @@ def test_fillna_frame(self, data_missing):
         # Non-scalar "scalar" values.
         super().test_fillna_frame(data_missing)
 
+    @pytest.mark.skip("Invalid test")
+    def test_fillna_fill_other(self, data):
+        # inplace update doesn't work correctly with patched extension arrays
+        # extract_array returns PandasArray, while dtype is a numpy dtype
+        super().test_fillna_fill_other(data_missing)
```

Comment:
This is a bit strange, but because I switched `if isinstance(.., Series): value = value._value` to `extract_array`, this test started to fail. That is because for `test_numpy.py` we patch the PandasArray to not be special cased, and then `extract_array` returns a PandasArray, even for a numpy dtype. And thus, in `iset` we set a PandasArray into a consolidated numpy block ..

Reply:
yah im not a fan of this patching, OK with xfailing